.changes/next-release/Bug Fix-e9afd2b9-cd47-4013-a2c0-8618d7b2ed15.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Change one of CodeWhisperer settings to User level settings"
```

This description is hard to understand.
```suggestion
	"description": "User setting for \"Share Code Whisperer Content With AWS\" was not reflected in workspace-level settings"
```